### PR TITLE
Introduce net-standard-table more accurately

### LINKED ISSUE #6022: Please update to lastest versions on tables etc.
The tables etc. are way outdated. Please update these along with the releases. MS Docs have become very confusing with lots of outdated documentation that no longer works or which works with what. If you want to have fast releases, the documentation has to keep up.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9eb8b774-c6f0-8235-6964-15219baf2ea7
* Version Independent ID: 6819a4a7-7527-d8c7-e132-767f7344e00e
* Content: [.NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
* Content Source: [docs/standard/net-standard.md](https://github.com/dotnet/docs/blob/master/docs/standard/net-standard.md)
* Product: **dotnet**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by rocifier**
## Summary

It was realised that some people not only miss the bullet points below the net-standard-table, but that the table was inaccurately introduced. Making the introduction to the table more verbose by pre-emptively projecting the purpose of the table to the reader and anticipating their confusion with missing data will help align their thoughts more clearly and free them to interpret the table using their own method or the guides below the table.

Fixes #6022

### CODE REVIEW

File: docs/standard/net-standard.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The various .NET implementations target specific versions of .NET Standard. Each
 
 ## .NET implementation support
 
-The following table lists all versions of .NET Standard and the platforms supported:
+The following table lists the minimum platform versions required when targeting each .NET Standard version.
```
Comment (svick) [inline docs/standard/net-standard.md]: I think it's not clear what "required" means here.

I would say something like:

> The following table lists the minimum platform versions that support each .NET Standard version.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you for contributing to the dotnet/docs repo, @rocifier, and for adding clarity to this introductory sentence. I agree with @svick, though, that  "that support" is clearer. Could you make that change?

**Comment by rocifier** [comment]
Hi, I have made the recommended change, thanks.

**Review by rpetrusha — APPROVED** [review]
Thanks, @rocifier, for making the additional changes. We'll merge your PR now. your changes should appear live on docs.microsoft.com in the next day or two.

**Pull request merged**